server/app/app.go

```diff
@@ -230,8 +230,8 @@ func (app *App) Start(opts ...appOption) error {
 
 	tracedbFactory := tracedb.Factory(tracesRepo)
 
-	eventEmitter := executor.NewEventEmitter(testDB, subscriptionManager)
-	registerOtlpServer(app, tracesRepo, runRepo, eventEmitter, dataStoreRepo, tracer)
+	// eventEmitter := executor.NewEventEmitter(testDB, subscriptionManager)
```

Comment:
We might need a flag to enable/disable this (?)

Reply:
Yes. If we cant have it on cloud, we should disable it via configuration